components/ReplyFeedback.js

```diff
@@ -291,28 +280,22 @@ function ReplyFeedback({
 
   return (
     <div className={cx(classes.root, className)}>
-      <button
-        className={cx(classes.vote, ownVote === 'UPVOTE' && classes.voted)}
-        type="button"
-        onClick={e => openVotePopover(e, 'UPVOTE')}
-        data-ga="Upvote"
-      >
-        <ThumbUpIcon className={classes.outlinedIcon} />
-      </button>
-      <button
-        className={cx(classes.vote, ownVote === 'DOWNVOTE' && classes.voted)}
-        type="button"
-        onClick={e => openVotePopover(e, 'DOWNVOTE')}
-        data-ga="Downvote"
-      >
-        <ThumbDownIcon className={classes.outlinedIcon} />
-      </button>
       <div className={classes.buttonGroup} data-ga="Number display">
-        <button className={classes.vote} type="button">
+        <button
+          className={cx(classes.vote, ownVote === 'UPVOTE' && classes.voted)}
+          type="button"
+          onClick={e => openVotePopover(e, 'UPVOTE')}
+          data-ga="Upvote"
+        >
           {positiveFeedbackCount}
           <ThumbUpIcon className={classes.thumbIcon} />
         </button>
-        <button className={classes.vote} type="button">
+        <button
+          className={cx(classes.vote, ownVote === 'DOWNVOTE' && classes.voted)}
+          type="button"
+          onClick={e => openVotePopover(e, 'DOWNVOTE')}
+          data-ga="Downvote"
```

Comment:
Since we block users from upvoting / downvoting their own reply, I think we should directly disable upvote / downvote button here, instead of displaying a popover that cannot submit.